### PR TITLE
Added MyDigipass.com

### PR DESCRIPTION
**Pull request by chanteux**

### CODE REVIEW

File: _data/providers.yml [hunk 1]
```diff
@@ -51,7 +58,6 @@ providers:
   - name: SAASPASS
     url: https://www.saaspass.com/
     img: saaspass.png
-    tfa: Yes
```
Comment (mxxcon) [inline _data/providers.yml:54]: Why did you delete this line?

File: _data/security.yml [hunk 2]
```diff
@@ -12,12 +12,28 @@ websites:
       tfa: No
       status: https://twitter.com/digicert/status/454722167897616384
 
+    - name: Dome9 Security
+      url: https://www.dome9.com/
```
Comment (mxxcon) [inline _data/security.yml]: Their site is mostly broken if you go to it through https. :/

**Comment by mxxcon** [comment]
Nexgate logo has to be 32x32.
So what exactly is mydigipass.com? Is it TFA provider or is it a password manager? And also how does it relate to Vasco?

File: _data/providers.yml (hunk 1 above)
Comment (chanteux) [inline _data/providers.yml:54]: This line is not needed for providers.

File: _data/security.yml (hunk 2 above)
Comment (chanteux) [inline _data/security.yml]: I see the same completely working site with http and https.

Comment (mxxcon) [inline _data/security.yml]: The site is not loading custom font they picked because of mixed content.
Linked screenshot on the main page results in 404.
"Schedule a Demo" results in 404.
"Feature Tour" results in 404.
"Pricing" results in 404.
etc etc etc

**Comment by chanteux** [comment]
The logo of nexgate is 32x32 pixels, please check again: https://raw.githubusercontent.com/chanteux/twofactorauth/9b6594e01d9732f86621e060b2374167c550d7c4/img/security/nexgate.png

Mydigipass.com is a tfa provider for website owners, offering a free password manager to all its users.

VASCO is the owner of Mydigipass.com

File: _data/security.yml (hunk 2 above)
Comment (chanteux) [inline _data/security.yml]: mmmh, your're right. 
I only checked the homepage...
I just sent an email to inform them.

Can you change the link to the Dome9 site to http?

Many thanks for your time!

Comment (mxxcon) [inline _data/security.yml]: @chanteux if you update that link to just `http://` in your repo's `patch-1` branch, it'll automagically update this pull request. :)

**Comment by mxxcon** [comment]
@chanteux if Vasco owns mydigipass, is there a reason to list them both?

**Comment by jdavis** [comment]
I get 404's for every single link on the Dome9 page :/ I won't merge because of that issue.

Nexgate looks fine though.

And I agree with @mxxcon. I was looking over Vasco's site and it looks like it includes all that MyDigiPass is. Since MyDigiPass is the managed service of Digipass, I'd say by adding just VASCO it would suffice. Basically VASCO is the "provider" in this case, MyDigiPass is just the service.

**Comment by jdavis** [comment]
Oh, and you can either clean up the history of this branch by creating a new branch and `cherry-pick`ing the commits and then doing `git push origin patch-1 --force`. Or you could open a new pull request.

One suggestion for the future, try to open a new branch/pull request for every change. In this case, we didn't agree with all of them so we can't merge the entire PR. If it had been in multiple pull requests, some of your stuff could be merged.

Just a little GitHub tip =]

Also, I'm sorry for being a stickler with the changes you've proposed. I am trying to keep a standard for adding sites and to keep the quality as high as possible. It's a difficult task as it requires things like this.

**Comment by chanteux** [comment]
@jdavis and @mxxcon thank you for your time and the great initiative for starting and maintaining this project.
It is indeed the first time for me... I changed this branch as you proposed.
Many thanks for the tips for the future ;-)

**Comment by mxxcon** [comment]
:+1:
Not the end of the world, but I still see Nexgate logo as 16x16. :/

**Comment by jdavis** [comment]
Once the logo is 32x32, I promise it will be merged, haha.

**Comment by jdavis** [comment]
@chanteux Strange. That failure isn't your fault. I'll do a manual merge of this to fix it.
